### PR TITLE
ci: add linter for markdown, yaml, CSS

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

**Comment by njzjz** [comment]
pre-commit.ci autofix

**Comment by njzjz** [comment]
pre-commit.ci autofix

File: doc/model/dplr.md [hunk 1]
```diff
@@ -157,19 +180,20 @@ Type 1 and 2 (O and H) are `real_atom`s, while type 3 (WCs) are `virtual_atom`s.
...
-
 ```
 fix ID group-ID style_name keyword value ...
 ```
-* ID, group-ID are documented in :doc:`fix <fix>` command
-* style_name = *dplr*
-* three or more keyword/value pairs may be appended
+
+- ID, group-ID are documented in :doc:`fix <fix>` command
+- style*name = \_dplr*
```
Comment (njzjz) [inline doc/model/dplr.md:195]:
```suggestion
- style\_name = _dplr_
```

- style\_name = _dplr_

**Comment by njzjz** [comment]
pre-commit.ci autofix

**Comment by njzjz** [comment]
https://github.com/deepmodeling/deepmd-kit/pull/3574/commits/fe17568b350604f93582fd854a2fa454ec517e0a seems wrong...

**Comment by njzjz** [comment]
> [fe17568](https://github.com/deepmodeling/deepmd-kit/commit/fe17568b350604f93582fd854a2fa454ec517e0a) seems wrong...

I skipped this line and submitted the issue to https://github.com/prettier/prettier/issues/16160.

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**